From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alex Bradbury <asb@lowrisc.org>
Subject: [RISCV] Implement RISCVRegisterInfo::enableMultipleCopyHints()

https://reviews.llvm.org/D38128 and r319754 introduced an opt-in register
allocator tweak with the stated intention of enabling it by default once
backends have all opted in. It doesn't seem to hurt us at least, so let's opt
in.
---
 test/CodeGen/RISCV/double-select-fcmp.ll | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/CodeGen/RISCV/double-select-fcmp.ll b/test/CodeGen/RISCV/double-select-fcmp.ll
index abb9ee5c8be..3a5ec81b44e 100644
--- a/test/CodeGen/RISCV/double-select-fcmp.ll
+++ b/test/CodeGen/RISCV/double-select-fcmp.ll
@@ -5,8 +5,8 @@
 define double @select_fcmp_false(double %a, double %b) nounwind {
 ; RV32IFD-LABEL: select_fcmp_false:
 ; RV32IFD:       # %bb.0:
-; RV32IFD-NEXT:    addi a0, a2, 0
 ; RV32IFD-NEXT:    addi a1, a3, 0
+; RV32IFD-NEXT:    addi a0, a2, 0
 ; RV32IFD-NEXT:    jalr zero, ra, 0
   %1 = fcmp false double %a, %b
   %2 = select i1 %1, double %a, double %b
-- 
2.17.2

